### PR TITLE
Add handler option for bubbles and keyboard buttons.

### PR DESCRIPTION
**Pull request by YunusovSamat**

### CODE REVIEW

**Review by lxmnk — APPROVED** [review]

File: botx/models/buttons.py [hunk 1]
```diff
@@ -22,6 +23,9 @@ class ButtonOptions(BotXBaseModel):
     #: text to be shown in toast (show command body if `alert_text` is `None`).
     alert_text: Optional[str] = None
 
+    #: if `client`, should be processed only by client.
```
Comment (Opti213) [inline botx/models/buttons.py:26]: platform, that handle message from markup. If `bot` command will come to bot, else(`client`) will be executed by client.

File: botx/models/enums.py [hunk 2]
```diff
@@ -141,3 +141,13 @@ class ClientPlatformEnum(str, Enum):
 
     #: Desktop platform.
     desktop = "desktop"
+
+
+class ButtonHandlerTypes(str, Enum):
+    """Enum for type of handler."""
```
Comment (Opti213) [inline botx/models/enums.py:147]: Enum for type of `handler` field for markup.

File: docs/changelog.md [hunk 3]
```diff
@@ -1,3 +1,11 @@
+## 0.17.1 (Feb 9, 2021)
+
+### Added
+
+* Add handler option for bubbles and keyboard buttons (`handler`). If `client`,
```
Comment (Opti213) [inline docs/changelog.md:5]: Add \`handler\` option for bubbles and keyboard buttons (`handler`). If `client`,

File: docs/changelog.md [hunk 4]
```diff
@@ -1,3 +1,11 @@
...
+  should be processed only by client
```
Comment (Opti213) [inline docs/changelog.md:6]: and message will not come to bot. If \`bot\` message will send by client to bot.

**Review by Opti213 — APPROVED** [review]

**Pull request merged**